<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>kds_main_info_print</class>
 <widget class="QDialog" name="kds_main_info_print">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>392</width>
    <height>136</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="lds_checkbox_phonestyle" name="checkBox">
     <property name="text">
      <string>启用</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1,0">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>打印机</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_printerpath">
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_printerset">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="fastfdmain_KDS/kds.qrc">
         <normaloff>:/kds/order_settings.png</normaloff>:/kds/order_settings.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="lds_toolbar_button" name="pushButton_ok">
       <property name="text">
        <string>确定</string>
       </property>
       <property name="outer_stylesheet" stdset="0">
        <string>pushbutton</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="lds_toolbar_button" name="pushButton_cancel">
       <property name="text">
        <string>取消</string>
       </property>
       <property name="outer_stylesheet" stdset="0">
        <string>pushbutton</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>lds_toolbar_button</class>
   <extends>QPushButton</extends>
   <header>lds_toolbar_button.h</header>
  </customwidget>
  <customwidget>
   <class>lds_checkbox_phonestyle</class>
   <extends>QPushButton</extends>
   <header>lds_checkbox_phonestyle.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="fastfdmain_KDS/kds.qrc"/>
 </resources>
 <connections/>
</ui>
